lib/ar_transaction_changes/version.rb

```diff
@@ -1,3 +1,3 @@
 module ArTransactionChanges
-  VERSION = "1.1.3"
+  VERSION = "1.2.0"
```

Comment:
Isn't this a backwards compatible change?  I think it should be `1.1.4`

Reply:
It's still backward compatible 
Changed version to `1.1.4`